run.sh

```diff
@@ -45,170 +45,338 @@ core install:
   skip-email: true
 EOF
 
+main() {
 
-# Download WordPress
-# ------------------
-if [ ! -f /app/wp-settings.php ]; then
-  printf "=> Downloading wordpress... "
-  chown -R www-data:www-data /app /var/www/html
-  sudo -u www-data wp core download >/dev/null 2>&1 || \
-    ERROR $LINENO "Failed to download wordpress"
-  printf "Done!\n"
-fi
+  # Download WordPress
+  # ------------------
+  if [ ! -f /app/wp-settings.php ]; then
+    h3 "Downloading wordpress"
+    chown -R www-data:www-data /app /var/www/html
+    WP core download |& loglevel
+    STATUS "${PIPESTATUS[0]}"
+  fi
 
+  # Wait for MySQL
+  # --------------
+  h2 "Waiting for MySQL to initialize..."
+  while ! mysqladmin ping --host=db --password="$DB_PASS" --silent; do
+    sleep 1
+  done
+
+  h1 "Begin WordPress Configuration"
+
+  h3 "Generating wp.config.php file"
+  rm -f /app/wp-config.php
+  WP core config |& loglevel
+  STATUS "${PIPESTATUS[0]}"
+
+  h2 "Checking database"
+  check_database
+
+  # Make multisite
+  # NOTE: This will likely cause issues down the road.
```

Comment:
Do you have something in mind? I have used multisites before and they don't defer that much from regular installs.